resources/tests.mjs

```diff
@@ -332,6 +332,52 @@ Suites.push({
     ],
 });
 
+Suites.push({
+    name: "NewsSite-Next",
+    url: "tentative/newssite/news-next/dist/index.html",
+    async prepare(page) {
+        await page.waitForElement("#close-toast-link");
+    },
+    tests: [
+        new BenchmarkTestStep("NavigateToUS", (page) => {
+            page.querySelector("#close-toast-link").click();
```

Comment:
We discussed this a bit before, but I have been considering if we should to force a layout in between each of these actions within a step. I believe this would create a more realistic workload for this type of test. While opening and immediately closing the navbar will create React and DOM work there won't be any layout required, while in practice a user discreetly clicking the toggle button twice would cause multiple layouts.

An alternative would be to make each step tiny (with a single action in sync time, then taking the forced layout in async time) but I understand there's a practical problem with noise when steps are so short. So I wonder if having a few big steps that run through a bunch of user actions would be a good compromise that allows them to be heavy enough to be more measurable but also not skipping work that would be happening if they were individual steps.

```
page.querySelector("#close-toast-link").click();
page.layout();
page.querySelector("#navbar-dropdown-toggle").click();
page.layout();
// and so on, possibly containing actions from multiple of the current BenchmarkTestSteps if there is a logical grouping
```

I'm curious for thoughts from others on the idea (cc @rniwa @camillobruni @smaug----)

Reply:
Forcing layouts after each click seems like a reasonable compromise to me since splitting work into multiple pieces can also lead to CPU getting throttled down.

Reply:
@bgrins - just to ensure I am making the right change - instead of multiple test steps, we'd have just one and calling the page.layout() function in between each action?

Reply:
added page.layout() function, but kept the individual steps for now. 
![Uploading Screenshot 2023-05-19 at 4.55.43 PM.png…]()

Reply:
Yes, something like that is what I was thinking